apps/shinyproxy/templates/deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         access: {{ .Values.permission }}
         app: shinyproxy-deployment
         project: {{ .Values.project.slug }}
+        site-dir: {{ .Values.appconfig.site_dir | trimAll "/" | replace "/" "-"  }}
```

Comment:
We should create a task that disallows any `paths/like\ this`

Reply:
Agreed